src/sagemaker/jumpstart/model.py

```diff
@@ -1036,13 +1036,15 @@ def _get_deployment_configs(
                 image_uri=image_uri,
                 region=self.region,
                 model_version=self.model_version,
+                hub_arn=self.hub_arn,
```

Comment:
can we add a unit test for this? seems like the current coverage didn't cover this bug

Reply:
synced with @evakravi offline, this needs to be covered through unit tests and I'll add it as a fast follow.